commander/src/commands/transaction/create/delegate.ts

```diff
@@ -74,15 +91,26 @@ export default class DelegateCommand extends BaseCommand {
 			},
 		} = this.parse(DelegateCommand);
 
-		const { username }: Args = args;
+		const { nonce, fee, username }: Args = args;
 		const networkIdentifier = getNetworkIdentifierWithInput(
 			networkIdentifierSource,
 			this.userConfig.api.network,
 		);
+
+		if (!isValidNonce(nonce)) {
+			throw new ValidationError('Enter a valid nonce in number string format.');
+		}
+
+		if (!isValidFee(fee)) {
```

Comment:
fee should be validated after the conversion, otherwise it fails on decimals

Reply:
It should fail on decimals because we expect a non-decimal input e.g. `transaction:create:delegate 1 100 lightcurve`

Reply:
when the input is `LSK` it should accept decimal points (it should be able to say `0.0001` LSK for fee